config/autoload/doctrine.local.external.php.dist

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
Why not name this `doctrine.external.php` instead of `doctrine.local.external.php`?